pango/src/layout.rs

```diff
@@ -0,0 +1,53 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::translate::*;
+
+use crate::LayoutLine;
+
+/// The result of [`LayoutLine::x_to_index`].
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+pub struct HitPosition {
```

Comment:
No need to re-declare this type, just re-use the other one.

Reply:
Well, once it's merged of course.

Reply:
Please just ignore this. I thought I was reviewing another PR of yours. Big brain time... :brain:

Reply:
sorry this is confusing, that PR includes the commit in #375, because it needs to also add custom methods in `layout.rs`.